src/shipit_dashboard/shipit_dashboard/api.py

```diff
@@ -164,8 +164,6 @@ def create_bug():
         link.roles = ','.join(user['roles'])
         db.session.add(link)
 
-        print('Contrib', contrib)
```

Comment:
i've added this issue to capture error like this, https://github.com/mozilla-releng/services/issues/96